src/Redux/Reducers/noteReducer.js

```diff
@@ -7,8 +7,8 @@ export const initialState = {
 export default function (state = initialState, action) {
   switch (action.type) {
     case actions.SET_NOTE:
-      let x = state.value;
-      let dataArray = [...x];
+      const newData = state.value;
```

Comment:
Hate to be annoying here but maybe this should be `existingData` as it is coming from the state and wouldn't be new data